packages/server/src/lib/email.js

```diff
@@ -337,6 +353,50 @@ async function sendGrantAssignedEmailsForAgency(assignee_agency, grantDetail, as
     await asyncBatch(inputs, deliverEmail, 2);
 }
 
+async function sendGrantSharedEmailsForAgency(assignee_agency, grant, grantDetail, assignorUserId) {
```

Comment:
One thing I've found a bit hard to follow in our codebase at times — and that I'm hoping we might be able to avoid introducing more of here — is when we partially update terminology in our code. (For example, is it "team" or "agency"? Are those synonyms or actually different concepts in our codebase?) 

It's totally natural to want to tweak user-facing language over time, but I'd love to see us try to maintain consistent naming conventions for a given concept in anything non-user-facing in our codebase. In general, that means when we make a user-facing terminology change like "assign" to "share", we should change the user-facing references but keep all internal code referencing "assign". (This assumes we're not interested in the effort of a full-scale naming change in the code, including things like database column renames, which is generally not worth it.)

Basically, I'd love to keep all internal code references as "assign", even as we're switching to calling it "share" with users. 

To put that into practice with this PR, that would mean naming this function `sendGrantAssignedEmailsForAgency` and renaming the existing function `sendGrantAssignedEmailsForAgencyDeprecated` or similar. (The second function should of course get cleaned up when we clean up the share terminology feature flag.)

Maybe I'm overly worried about such things, but I've personally been thrown off by this sort of thing in our codebase before. And these paper cuts tend to add up over time in a codebase. If you disagree and don't think we need to sweat this stuff, feel free to disregard — I'm just one volunteer in the codebase, not the arbiter of such things. :)

Reply:
Made the change!